src/expression/context.ts

```diff
@@ -1,6 +1,7 @@
-import { ExpressionNode, ExpressionNodeType } from './node'
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ExpressionContext {}
 
-export interface ExpressionContext {
-  expressionNode?: ExpressionNode<ExpressionNodeType>
+export interface ExpressionNodeContext {
+  object?: any
   evaluateToNode?: boolean
 }
```

Comment:
lets have only one context. and I will still call expressionNode even if it's type is any.

Reply:
ok